### ISSUE TITLE
Animations look stiff when using Rokoko to retarget animations.

### ISSUE THREAD

**Issue by BeezBumba** [niacdoial/blemd#89]
I am using rokoko to retarget a model im using for a hack. The result animation looks like it's still in the t-pose? I'm pretty sure that Rokoko works well, because I used it for many other armatures that worked. Is there a way to solve this? The image is at frame 39. I am trying to get the issue out quick so this is why I not uploading a video, if you need a video I can send you one!
![Screenshot 2024-07-23 153033](https://github.com/user-attachments/assets/131d2a2d-58d2-47cc-a366-ff26a7e028f4)

**Comment by niacdoial** [comment]
yeah I think I'll have to look at the video to see if anything is clearer.
also, why do you have another model on the right? is it what the pose is supposed to look like at this frame?
